api-report/sequence.api.md

```diff
@@ -193,18 +193,18 @@ export class Interval implements ISerializableInterval {
     getIntervalId(): string;
     // (undocumented)
     getProperties(): PropertySet;
-    // @deprecated
+    // @internal
```

Comment:
Hmmmm I suppose it is not.. IMO the breaking change would result in an error in `test/types/validateXXXPrevious.generated.ts` when you run the build command, and requires you to update the `typeValidation.broken` entry in `package.json`. I did not meet this error in this case. Probably need some clarification from @Abe27342

Reply:
we should avoid doing it. It won't break customers on its own, but future changes to the same branch might: say that I see `union` is marked internal and decide I want to modify/remove it in main which I should be free to do if customers followed our notions of internal types/methods. This could cause a breaking change in a minor version.

That said, there is basically no validation currently that customers don't use internal types or methods. There's a clear path forward to making sure they don't use entire types marked internal, as API-Extractor supports removing them from the .d.ts rollup (at which point it'd be a compile error). I don't think there's any analogous support for trimming internal methods on an otherwise public interface though.

A breaking change is better defined as one that could break compilation or execution of reasonable usage code: the type validation we have is just a tool, not end-all-be-all.

Reply:
Have removed the change of interface tags (as well as the changeset), will include them in the PR merged to next